Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -82,24 +82,30 @@ size_t Sheet::add_row()
     return m_rows++;
 }
 
-static size_t convert_from_string(StringView str, unsigned base = 26, StringView map = {})
+static constexpr size_t convert_from_string(StringView str)
 {
-    if (map.is_null())
-        map = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-
-    VERIFY(base >= 2 && base <= map.length());
-
     size_t value = 0;
-    for (size_t i = str.length(); i > 0; --i) {
-        auto digit_value = map.find(str[i - 1]).value_or(0);
+    const auto len = str.length();
+    for (auto i = 0u; i < len; i++) {
+        assert(str[i] >= 'A' && str[i] <= 'Z');
```

Comment:
Please use `VERIFY()` instead of `assert()` here.

and I'm still against changing the behaviour of this function past fixing the actual bug.

Reply:
I will add back `base` and `map`.